signatures/slot.yml

```diff
@@ -115,3 +118,26 @@ Slot:
         type: Object[]
         description: Successfully reset slot
     errors: {}
+  AdminSlotReset:
+    path: /slot/admin/reset
+    method: POST
+    title: Admin reset Slot availablity
+    description: reset a slot
+    params:
+      hospital:
```

Comment:
Do you mind adding "hospital_name" and "disease_name", as String fields in the response? We are about to add these two fields to save some API calls for the frontend.